includes/setup-wizard.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Setup Wizard Class
+ *
+ * Extends the original WC setup wizard so we can add in new templates to the view.
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
```

Comment:
Perhaps I'm being paranoid, since this should _never_ happen since this plugin won't load unless Woo is installed, but should we verify that the `WC_Admin_Setup_Wizard` class exists here too and exit if not?

Reply:
That seems reasonable.  I was contemplating whether or not we needed this since we'll presumably check for dependencies prior to this (actually looks like we currently don't have any dependency checks yet).

We can err on the side of caution here though.